llpc/docs/DdnPackShaderInputOutput.md

```diff
@@ -74,7 +74,7 @@ layout(location = 7) in float16_t v8;
                        |                            (3,1,false) <-> (2,2,true)
                        |                                      |
                        |``````````````````````````````````````|
-            (Ressemble output export)               (Fill InOutLocMap with AsIndex() of origin and new)
+            (Resemble output export)               (Fill InOutLocMap with AsIndex() of origin and new)
```

Comment:
Not sure if this should be `Resemble` or `Reassemble`.

Reply:
I think it's 'reassemble', because it's trying to recreate in-out mapping after some optimizations.